apps/core/app/[locale]/(default)/product/[slug]/_components/reviews.tsx

```diff
@@ -1,24 +1,64 @@
+import { removeEdgesAndNodes } from '@bigcommerce/catalyst-client';
 import { getFormatter, getTranslations } from 'next-intl/server';
 
-import { getProductReviews } from '~/client/queries/get-product-reviews';
+import { client } from '~/client';
+import { graphql } from '~/client/graphql';
+import { revalidate } from '~/client/revalidate-target';
 import { Rating } from '~/components/ui/rating';
 
-import { ProductReviewSchema } from './product-review-schema';
+import { ProductReviewSchema, ProductReviewSchemaFragment } from './product-review-schema';
+
+export const ReviewsQuery = graphql(
```

Comment:
This is a query and not a fragment as this component is wrapped in `<Suspense>` needs to do its own request.